src/controllers/routers/beneficiaries.js

```diff
@@ -126,6 +129,45 @@ const getBeneficiary = async (req, res, next) => {
   }
 };
 
+/**
+ * Retrieve related cases by id
+ * @param {Request} req
+ * @param {Response} res
+ */
+const getBeneficiaryCases = async (req, res, next) => {
+  const { id } = req.params;
+  try {
+    // get list of case numbers from Case table based on beneficiaryId
+    const caseNumber = await Case.query()
```

Comment:
update the var name to avoid having both caseNumber and caseNumbers.
you could change this to results.

Reply:
changed var name from caseNumber to results